scipy-1.0/core-dev.tex

```diff
@@ -0,0 +1,66 @@
+\author[5]{Eric Jones}
+\author[5]{Robert Kern}
+\author[21]{Eric Larson}
+\author[22]{CJ Carey}
+\author[10]{\.{I}lhan Polat}
+\author[23]{Yu Feng}
+\author[24]{Eric W. Moore}
+\author[25]{Jake VanderPlas}
+\author[10]{Denis Laxalde}
+\author[10]{Josef Perktold}
+\author[26]{Robert Cimrman}
+\author[27]{Ian Henriksen}
+\author[10]{E. A. Quintero}
+\author[10]{Charles R Harris}
+\author[28]{Anne M. Archibald}
+\author[29]{Ant\^{o}nio H. Ribeiro}
+\author[30]{Fabian Pedregosa}
+\author[30]{Paul van Mulbregt}
+\author[ ]{SciPy 1.0 Contributors}
+\affil[1]{University of Jyv\"askyl\"a, FI-40014 University of Jyv\"askyl\"a, Finland}
+\affil[2]{Quansight LLC, Austin, TX, USA}
+\affil[3]{Ultrasound Imaging, Mayo Clinic, Rochester, MN 55902, USA}
+\affil[4]{Electrical Engineering, Brigham Young University, Provo, UT 84604, USA}
+\affil[5]{Enthought, Inc., Austin, TX 78701, USA}
+\affil[6]{Anaconda Inc, Austin, TX 78701, USA}
+\affil[7]{BioResource and Agricultural Engineering Department, California Polytechnic State University, San Luis Obispo, CA 93407, USA}
+\affil[8]{Department of Mathematics, University of California, Los Angeles, CA 90025, USA}
+\affil[9]{Los Alamos National Laboratory, Los Alamos, NM 87544}
+\affil[10]{Independent Researcher}
+\affil[11]{Higher School of Economics, Myasnitskaya 20, Moscow 101000 Russia}
+\affil[12]{Independent Researcher, Saue, Estonia}
+\affil[13]{Department of Mechanics and Applied Mathematics, Institute of Cybernetics at Tallinn Technical University, Akadeemia Rd 21 Tallinn, Estonia}
+\affil[14]{Independent Researcher, NY, USA}
+\affil[15]{Berkeley Institute for Data Science, University of California Berkeley, Berkeley, CA 94720 USA}
+\affil[16]{School of Psychology, University of Birmingham, Edgbaston, Birmigham B15 2TT, UK}
+\affil[17]{Berkeley Institute for Data Science, University of California, Berkeley, CA 94720 USA}
+\affil[18]{Division of Biostatistics, University of California, Berkeley, CA 94720, USA}
+\affil[19]{WayRay LLC, Skolkovo Innovation Center, Moscow, 143026, Russia}
+\affil[20]{Australian Nuclear Science and Technology Organisation, Locked Bag 2001, Kirrawee DC, NSW 2232, Australia}
+\affil[21]{Institute for Learning and Brain Sciences, University of Washington, Seattle, WA 98195, USA}
+\affil[22]{College of Information and Computing Sciences, University of Massachusetts Amherst, Amherst, MA 01002, USA}
+\affil[23]{Berkeley Center for Cosmological Physics, University of California, Berkeley, CA 94720, USA}
+\affil[24]{Bruker Biospin Corp., 15 Fortune Drive, Billerica, MA 01821}
+\affil[25]{University of Washington, Seattle, WA 98195, USA}
+\affil[26]{New Technologies - Research Centre, University of West Bohemia, Univerzitní 8, 306 14, Plzeň, Czech Republic}
+\affil[27]{Oden Institute for Computational Engineering and Sciences, The University of Texas at Austin, Austin, Texas 78712-1229, USA}
+\affil[28]{Anton Pannekoek Institute, P.O. Box 94249, 1090 GE Amsterdam, Nederland}
```

Comment:
This seems to be the only affiliation with the country name in the local language, perhaps change to "the Netherlands" for consistency.

Reply:
Will do

Reply:
Ok, I fixed this one